src/fluree/store/proto.cljc

```diff
@@ -0,0 +1,8 @@
+(ns fluree.store.proto
```

Comment:
In the interest of getting https://github.com/fluree/core/issues/57 and https://github.com/fluree/core/issues/58 shipped, I'd like to defer this work to when we fix the IPFS conn, since that will involve additional refinements to the Store protocol.